src/HBL01MItineraryHashtag/HBL01MItineraryHashtag.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Get } from '@nestjs/common';
+
+import { HBL01MItineraryHashtagService } from './HBL01MItineraryHashtag.service';
+
+@Controller('HBL01')
+export class HBL01MItineraryHashtagController {
+  constructor(private readonly service: HBL01MItineraryHashtagService) {}
+
+  // curl -XGET -H "Content-Type:application/json" localhost:3000/HBL01
```

Comment:
ゴメン、ポート番号衝突しちゃうから、3072に変えた
```suggestion
  // curl -XPOST -H "Content-Type:application/json" localhost:3072/HBL001
```

Reply:
[f5d4cbb](https://github.com/Ayato-kosaka/spelieve-backend-tag/pull/3/commits/f5d4cbbd45aeb7bdd667adea59e748a33406ac6a)で対応